docs/examples/pv_subdir.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
If we're providing examples for a PV now (a good thing), then we should make sure to update the customer-facing docs with instructions for doing this.

Reply:
Noted for documentation